provider/cmd/app/app.go

```diff
@@ -60,15 +66,18 @@ func init() {
 }
 
 type Options struct {
-	Address string
+	Address          string
+	StreamingAddress string
+	BaseURL          string
+
 	RootDir string
 
 	PathSupportedMachineClasses string
 
 	ApinetKubeconfig string
 
-	Libvirt   LibvirtOptions
-	NicPlugin *networkinterfaceplugin.Options
+	LibvirtOpts LibvirtOptions
```

Comment:
pls keep `Libvirt` because it is clear that we are talking about options and it reads much better `options.Libvirt.Socket`